test/extension-tests/single-root-UI/preset-v5-tests/project-folder/CMakePresets.json

```diff
@@ -0,0 +1,60 @@
+{
+  "version": 5,
+  "configurePresets": [
+    {
+      "name": "Linux1",
+      "description": "Invalid preset for < v5 presets using pathListSep macro",
+      "generator": "Ninja",
+      "binaryDir": "${workspaceFolder}/build",
+      "cacheVariables": {
+        "CMAKE_BUILD_TYPE": "Debug",
+        "CMAKE_TOOLCHAIN_FILE": {
+            "type": "FILEPATH",
+            "value": "${workspaceFolder}/test-toolchain.cmake"
+        }
+      },
+      "environment": {
+        "MY_ENV": "$penv{PATH}${pathListSep}${workspaceFolder}/build"
```

Comment:
For this test, I think it would be interesting to check that the separator is replaced properly. I didn't see where this environment variable was consumed and I didn't see any test that validated it.